.gitignore

```diff
@@ -7,7 +7,7 @@ npm-debug.log
 coverage/
 .linkinator
 /assets/images/early-access
-/assets/fonts/inter
+# /assets/fonts/inter
```

Comment:
I would delete instead of commenting out, across the pull request. We always have the old code in version history if needed

Reply:
Thank you! Deleted the stuff!